src/fastcs/control_system.py

```
@@ -155,14 +156,15 @@ async def block_forever():
 
     async def _interactive_shell(self, context: dict[str, Any]):
         """Spawn interactive shell in another thread and wait for it to complete."""
+        loop = asyncio.get_running_loop()
 
         def run(coro: Coroutine[None, None, None]):
             """Run coroutine on FastCS event loop from IPython thread."""
 
             def wrapper():
                 asyncio.create_task(coro)
```

Ruff (0.15.5)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)